src/routes/(user)/clients/addClient/+page.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+	import { goto } from '$app/navigation';
+    import type { PageData } from './$types';
+    import { superForm } from 'sveltekit-superforms';
+    import SuperDebug from 'sveltekit-superforms';
+    export let data : PageData;
+
+    const { form } = superForm(data.form);
+
+   
+
+
+    const goToClient = () => {
+		goto('../clients');
+	};
+</script>
+
+
+<section class="flex-1 space-y-4 p-8 pt-6">
+	<div class="flex items-center justify-between space-y-2">
+		<h2 class="text-3xl font-bold tracking-tight">Add Client</h2>
+		<button type="submit" class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded" on:click={goToClient}>Back to all clients</button>
```

Comment:
This should be a normal button. We can replace this with the shadcn-svelte provided Button component.